index.html

```diff
@@ -1,14 +1,100 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <title>Mockup Home</title>
+    <title>Colorado</title>
+    <link href="style.css" rel="stylesheet"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Giga&display=swap" rel="stylesheet"/>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+  <body> 
+   <div id="most"> 
+        <div id="margin1">margin1</div>
+        <div id="margin2">margin2</div>
+        <div id="margin3">margin3</div>
+        <div id="margin4">margin4</div>

```

Comment:
Using grid there is a way to say "this grid square is empty" rather than having empty `div` elements that you assign to those spaces. The way to do this is in your `grid-template-area` you can say that the section you want to be empty is assigned by a '.'